docs/tutorials/acme/dns-validation.rst

```diff
@@ -86,30 +75,45 @@ Once we have created the above Issuer we can use it to obtain a certificate.
 .. code-block:: yaml
    :linenos:
 
-   apiVersion: certmanager.k8s.io/v1alpha1
-   kind: Certificate
-   metadata:
-     name: example-com
-     namespace: default
-   spec:
-     secretName: example-com-tls
-     issuerRef:
-       name: letsencrypt-staging
-     commonName: '*.example.com'
-     dnsNames:
-     - example.com
-     - foo.com
-     acme:
-       config:
-       - dns01:
-           provider: prod-dns
-         domains:
-         - '*.example.com'
-         - example.com
-       - dns01:
-           provider: cf-dns
-         domains:
-         - foo.com
```

Comment:
The old example explicitly shows how you can configure two different DNS providers for a single certificate (`foo.com` uses a cloudflare solver).

Can you add an additional `solver` entry to the Issuer that has the following selector:

```
selector:
  dnsNames:
  - foo.com
```

Reply:
sure. I add it on Monday. thx for review

Reply:
added